client/src/amplify-ui/cli-feature-flag/feature-flags.json

```diff
@@ -143,63 +143,63 @@
     "description": "Feature Flag related to Codegen",
     "features": {
       "useAppsyncModelgenPlugin": {
-        "description": "Use the migrated implementation of internal DataStore model generator package at https://github.com/aws-amplify/amplify-codegen",
+        "description": "Choose between implementations of internal DataStore model generator package. Refer [link to FeatureFlags.md]",
```

Comment:
here and below: Does `[link to FeatureFlags.md]` get auto-replaced, or do you need to actually link it? :)

Reply:
Added the link